app/build.gradle

```diff
@@ -47,9 +47,12 @@ android {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            buildConfigField "int", "MaxDayBeforeDeletingSentSurvey", "14"
+
```

Comment:
@Amejia481 this applies to all variants, so if we accept this code, all variants would suddenly start deleting sent surveys older than 14. I think the beauty of including the gradle variable would be to do it only for target variant (ereferrals). Maybe you can change it into a boolean var deciding if the variant must performMaintenanceTasks and apply the surveys removal only if that variable is true

Reply:
You are correct, I made the change.